### PR TITLE
added a keyboard shortcut n for directly jumping for creating a new database

### PR DESCRIPTION
**Pull request by akshilshah**
added a keyboard shortcut n for directly jumping for creating a new database

Signed-off-by: Akshil shah <akshilshah4444@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

File: js/vendor/jquery/jquery-migrate.js [hunk 1]
```diff
@@ -1,626 +1,215 @@
...
-
-	return deferred;
-};
-
-// Preserve handler of uncaught exceptions in promise chains
-jQuery.Deferred.exceptionHook = oldDeferred.exceptionHook;
-
-return jQuery;
-} );
+/*! jQuery Migrate v3.0.1 | (c) jQuery Foundation and other contributors | jquery.org/license */
```
Comment (mauriciofauth) [inline js/vendor/jquery/jquery-migrate.js]: Hi, @akshil4444. Updating jQuery Migrate does not seem to be related to this PR. Could you remove this change? Thank you.

File: js/shortcuts_handler.js [hunk 2]
```diff
@@ -96,6 +97,8 @@ $(document).ready(function () {
             window.history.back();
         } else if (e.keyCode === keyH) {
             $('.ic_b_home').first().trigger('click');
+        } else if (e.keyCode === keyN) {
+            window.open('http://localhost/phpmyadmin/server_databases.php?server=1', '_self');
```
Comment (ibennetch) [inline js/shortcuts_handler.js]: This takes me to an error page that says "Unable to connect. Firefox can’t establish a connection to the server at localhost."

**Review by ibennetch — CHANGES_REQUESTED** [review]

**Comment by ibennetch** [comment]
I'm not sure how we want to handle shortcut keys; this may not be the best way (for instance, the last time this came up we discussed how "N" for "new" makes sense in English, but not for all other languages where the word "new" translates to something that doesn't start with an 'n'.

File: js/shortcuts_handler.js (hunk 2 above)
Comment (akshil4444) [reply js/shortcuts_handler.js]: confirmed, working on this.

Comment (akshil4444) [reply js/shortcuts_handler.js]: @ibennetch The firefox error gives only for the first time and it's actually not an error as it is working.  After that, if you try to do the same there won't be any error. I actually don't know for what reason is the error occurred for the initial time.

Comment (saulotoledo) [reply js/shortcuts_handler.js]: @akshil4444 I suggest to change the port of your local HTTP server and test again.

**Pull request closed without merging**